test/JDBC/upgrade/14_6/schedule

```diff
@@ -242,7 +242,7 @@ BABEL-SPCOLUMNS
 BABEL-SP_TABLES
 BABEL-SP_SPECIAL_COLUMNS
 BABEL-SP_TABLE_PRIVILIGES
-BABEL-SP_FKEYS
+# BABEL-SP_FKEYS
```

Comment:
Add a comment with jira ID here similar to https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_3_X_DEV/test/JDBC/upgrade/latest/schedule#L249